.eslintrc

```diff
@@ -1,4 +1,7 @@
 {
   "plugins": ["jest"],
-  "extends": ["standard", "plugin:jest/recommended"]
+  "extends": "@adobe/eslint-config-aio-lib-config",
+  "rules": {
+      "node/no-unpublished-require": 0
```

Comment:
no, this shouldn't be here globally but only in tests or e2e folder. we want to know if an unpublished require is in the code, because our code will crash since we didn't include the package.